src/main/java/uk/ac/ebi/embl/gff3tools/gff3/GFF3Anthology.java

```diff
@@ -14,17 +14,19 @@
 
 public class GFF3Anthology {
```

Comment:
We should deprecate this file. Do not add new terms to it. Ideally I want to remove it after we implement all the validations.